andela/suyabay
# [#116785861] Create /developer/myapp/new page where users can register their app

#### What does this PR do?

Create a page where user can register their application
#### Description of Task to be completed?

/developers/myapp/new route presents a page with a simple form to fill in all the required details like Application name, Homepage Url. Application Description and a Register Application button.
#### How should this be manually tested?

From /developer/myapp page, click on Create A New App button. This takes you to /developer/myapp/new which contains the simple form. See the below to view the screenshot

 #### Any background context you want to provide?

None
#### What are the relevant pivotal tracker stories?
#116785861
#### Screenshots (if appropriate)

![screen shot 2016-04-05 at 11 16 55 am](https://cloud.githubusercontent.com/assets/14159420/14278512/04f0913a-fb20-11e5-8d32-759e13e0c8cd.png)
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
